### PR TITLE
Add redux

### PR DESCRIPTION
**Pull request by Luchy-B**
- Add styling
- Fetch data using API
- Add linters

### CODE REVIEW

File: src/Test/Slice.test.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { countriesReducer } from '../Redux/Countries/countriesSlice';
+
+test('should return the initial state', () => {
+  expect(countriesReducer(undefined, { type: undefined })).toEqual(
+    {
+      countries: [], status: false, error: false,
+    },
```
Comment (hiromataba) [inline src/Test/Slice.test.js:7]: - [ ] Please add more tests for you components as required 

image

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import Countries from './components/Countries';
+import Footer from './components/Footer';
+
+function App() {
+  return (
+    <div className="App">
+      <Countries />
+      <Footer />
+    </div>
+  );
```
Comment (hiromataba) [inline src/App.js:10]: - [ ] Please fix your mobile design to match the orginal one as required 

- Orginial design 👇 

image

- Your design 👇 

image

image

File: README.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank Microverse for Guiding me through this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (hiromataba) [inline README.md]: - [ ] You should give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @Luchy-B    :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title and description  :+1: 
- Your linters are passing :heavy_check_mark:  
- Profesional README file :rocket: 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 2 above)
Comment (t-yanick) [reply src/App.js:10]: - [ ] Good job working on most of the requirements requested by the previous reviewer. Unfortunately the country details page I think needs a right margin. As you can see, the Incident Rate cell aligns with the container. Kindly give some margin to make your design similar. See screenshot below.
Screen Shot 2023-04-28 at 9 20 59 PM

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @Luchy-B , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amejid — APPROVED** [review]
Hi @Luchy-B, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct workflow :heavy_check_mark: 
- Functionalities working as expected :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**